Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -34,6 +34,8 @@ list of important variables.
    | AMREX_HOME | Path to amrex                       | environment |
    +------------+-------------------------------------+-------------+
    | COMP       | gnu, cray, ibm, intel, llvm, or pgi | none        |
+   +------------|-------------------------------------+-------------+
+   | CXXSTD     | C++ Standard (11, 14, 17, ...)      | 11          |
```

Comment:
we could also write the whole string, e.g. `c++11`, and `gnu11` here to be able to express compiler extensions. what do you think?

Reply:
I actually like `c++11` better than `11` for the reason that we can set `gnu++`.  I made the comment because the code is currently doing CXXFLAGS += -std=c++$(CXXSTD).

Reply:
Ok cool, then I'll change this as indicated because it's more general.